airbyte-webapp/src/core/domain/job/Job.ts

```diff
@@ -38,6 +77,21 @@ export interface JobDetails {
   attempts: AttemptInfo[];
 }
 
+export interface JobDebugInfoMeta {
+  airbyteVersion: string;
+  id: number | string;
```

Comment:
When can this be `string`? According to the specification this is of `type: integer`?